libs/network/esp8266/jswrap_esp8266.c

```diff
@@ -354,10 +411,10 @@ void jswrap_ESP8266WiFi_onWiFiEvent(
   ) {
   // If the callback is null
   if (callback == NULL || jsvIsNull(callback)) {
-    if (jsWiFiEventCallback != NULL) {
-      jsvUnLock(jsWiFiEventCallback);
+    if (g_jsWiFiEventCallback != NULL) {
+      jsvUnLock(g_jsWiFiEventCallback);
```

Comment:
jsvUnLock is null-safe, so here (and in other places) you don't actually need the explicit pointer check - not that it does much harm though